ui/config/webpack.config.js

```diff
@@ -495,6 +501,34 @@ module.exports = function(webpackEnv) {
           {
             inject: true,
             template: paths.appHtml,
+            excludeChunks: [ 'list' ]
```

Comment:
This is the include for the reader / main application, so we exclude the `list` bundle.